modules.d/95virtiofs/mount-virtiofs.sh

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/sh
+
+virtiofs_fstype_opt() {
+	[ -n "$fstype" -a "$fstype" = "virtiofs" ]
```

Comment:
It's not necessary to test `$fstype` for non-emptiness if you then compare it with a concrete string, is it?